File: go/iq/newsletter~set~live_updates.go
// BuildNewslettersSubscribeToLiveUpdates builds <iq type="set" xmlns="newsletter">. Replies with one of: Success, ClientError, ServerError.
func BuildNewslettersSubscribeToLiveUpdates(to types.JID, to types.JID) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"to": to, "xmlns": "newsletter", "type": "set"},
		Content: []waBinary.Node{
			{Tag: "live_updates", Attrs: waBinary.Attrs{}},
		},
	}
}

File: ts/iq/newsletter~set~live_updates.ts
/** <iq type="set" xmlns="newsletter">. Replies with one of: Success, ClientError, ServerError. */
export function buildNewslettersSubscribeToLiveUpdates(to: string, to: string): BinaryNode {
  return { tag: 'iq', attrs: { to: to, xmlns: 'newsletter', type: 'set' }, content: [
      { tag: 'live_updates', attrs: {  } },
    ] }
}